pinecone/manage.py

```diff
@@ -201,7 +201,7 @@ def describe_index(name: str):
     return IndexDescription(name=db['name'], metric=db['metric'],
                             replicas=db['replicas'], dimension=db['dimension'], shards=db['shards'],
                             pods=db.get('pods', db['shards'] * db['replicas']), pod_type=db.get('pod_type', 'p1'),
-                            index_config=db['index_config'], status={'ready': ready, 'state': state},
+                            index_config=db.get('index_config'), status={'ready': ready, 'state': state},
```

Comment:
This may still show up as None in the response. Should we just get rid of it?

Reply:
good point, let's just remove